edgexfoundry/edgex-go
# fix: Fix core command swagger

**If your build fails**  due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/edgex-go/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

  <link to docs PR>

## Testing Instructions
following https://github.com/edgexfoundry/edgex-go/pull/4569 test instructions for fuzzing test, you will observe the fuzzing test coverage has increased for core-command after this fix.

## New Dependency Instructions (If applicable)
